flutter/lib/src/integrations/mobile_vitals_integration.dart

```diff
@@ -72,6 +65,28 @@ class _NativeAppStartEventProcessor extends EventProcessor {
   }
 }
 
+class _NativeFramesEventProcessor extends EventProcessor {
+  _NativeFramesEventProcessor(this._native);
+
+  final SentryNative _native;
+
+  @override
+  FutureOr<SentryEvent?> apply(SentryEvent event, {hint}) async {
+    if (event is SentryTransaction) {
+      final traceId = event.contexts.trace?.traceId;
```

Comment:
Yeah, on the other hand when accessing the `ISentryFrame` objects form other places, we always have to access through `contexts.trace?.traceId`, so it makes sense that this reads the same, no?

Reply:
`contexts.trace` is built out from the `transaction.eventId` anyway, I just thought about it because the eventId is not nullable, but your call here, I'm fine either way, just a nit.

Reply:
Obsolete, as we are now using `SentryTracer` instead.